docs/tests.rst

```diff
@@ -110,6 +110,57 @@ Via Docker with a graphical instance of Firefox (macOS)
 
   $ docker-compose -f docker-compose.test.yml run --rm acceptance
 
+How to Debug the Selenium Tests on Docker
+-----------------------------------------
+In order to debug code in the `web` container with the Docker-Selenium test suite it is necessary to use `remote-pdb`_.
+
+1. First install RemotePdb in the web container. Run the web container with the bash command attached:
+
+.. code-block:: bash
+
+  $ docker-compose run web bash
+
+Secondly install with the pip command the `remotePdb` package:
+
+.. code-block:: bash
+
+  (virtualenv) $ pip install remote-pdb
+
+Finally exit from the virtualenv and from the docker container with `ctrl + d`.
```

Comment:
All this first paragraph can be replaced with `docker-compose run --rm web pip install remote-pdb`.